Documentation/Books/Manual/Views/ArangoSearch/GettingStarted.md

```diff
@@ -0,0 +1,107 @@
+# Getting started with ArangoSearch views
+
+## The DDL configuration
```

Comment:
Could use an explanation what DDL stands for (maybe add a brief overview of the steps above this headline?)